src/Uno.UI.Runtime.Skia.X11/X11PointerInputSource.cs

```diff
@@ -188,7 +188,7 @@ private PointerPoint CreatePointFromCurrentState(IntPtr time)
 		// This matches the format of WinUI. See also: https://github.com/unoplatform/uno/issues/14535
 		var point = new PointerPoint(
 			frameId: (uint)time, // UNO TODO: How should set the frame, timestamp may overflow.
-			timestamp: (uint)time,
+			timestamp: (ulong)(time * TimeSpan.TicksPerMillisecond),
```

Comment:
Is there a manual test we can use to validate the change?

Reply:
Indeed, working to add a test for inertia for https://github.com/unoplatform/uno/pull/18294, will also flag it as manual to validate ensure there is no platform specific regression like this.